src/extension.spec.ts

```diff
@@ -58,6 +57,7 @@ vi.mock('./macadam-machine-stream.js', async () => {
 
 beforeEach(() => {
   vi.resetAllMocks();
+  vol.reset();
```

Comment:
Wondering if we should not have inside the `setup-vitest` this reset logic?

Reply:
We are used to see resetAllMocks in `beforeEach` for each file, I would prefer to also have `vol.reset` which is very similar